modules/rosa-hcp-classic-comparison.adoc

```diff
@@ -57,6 +57,8 @@
 * US East - N. Virginia (us-east-1)
 * US East - Ohio (us-east-2)
 * US West - Oregon (us-west-2)
```

Comment:
Hi @EricPonvelle .

In the ROSA service defintion document, the Regions and availability zones table references 

- ap-northeast-1 (Tokyo)
- ap-northeast-2 (Seoul) x 2

Should these entries belong in this file as HCP is the target medium.

Reply:
There was a merge conflict. It should be corrected now.

Reply:
I still see teh entries in the provided preview link for [Regions and availability zones](http://file.rdu.redhat.com/eponvell/OSDOCS-9524_ap-northeast-2/rosa_architecture/rosa_policy_service_definition/rosa-service-definition.html#rosa-sdpolicy-regions-az_rosa-service-definition) for ROSA Classic. Maybe the preview link needs a refresh on the PR (happens to me quiet a lot)?